cloud-consumption/src/main/java/com/sequenceiq/consumption/flow/consumption/storage/event/SendStorageConsumptionEvent.java

```diff
@@ -10,33 +12,34 @@
 
 public class SendStorageConsumptionEvent extends StorageConsumptionCollectionEvent {
 
-    private final StorageConsumptionResult storageConsumptionResult;
+    private final Set<StorageConsumptionResult> storageConsumptionResults;
 
-    public SendStorageConsumptionEvent(String selector, Long resourceId, String resourceCrn, StorageConsumptionResult storageConsumptionResult) {
+    public SendStorageConsumptionEvent(String selector, Long resourceId, String resourceCrn,
+        Set<StorageConsumptionResult> storageConsumptionResults) {
         super(selector, resourceId, resourceCrn);
-        this.storageConsumptionResult = storageConsumptionResult;
+        this.storageConsumptionResults = storageConsumptionResults;
     }
 
     @JsonCreator
     public SendStorageConsumptionEvent(
             @JsonProperty("selector") String selector,
             @JsonProperty("resourceId") Long resourceId,
             @JsonProperty("resourceCrn") String resourceCrn,
-            @JsonProperty("storageConsumptionResult") StorageConsumptionResult storageConsumptionResult,
+            @JsonProperty("storageConsumptionResults") Set<StorageConsumptionResult> storageConsumptionResults,
```

Comment:
It will break existing payloads, this parameter's value will be null, to be more precise